File: Python Assignments/untitled folder/guessTheNumberGame.java
import java.util.Scanner; 

public class guessTheNumberGame{
    public static void main(String[] args){
        int secretNumber = 7;
        boolean incorrectGuess = true;
        Scanner userInput = new Scanner(System.in);

        while(incorrectGuess){
            System.out.print("Guess!, enter a number: ");
            int userGuess = userInput.nextInt();
            if (userGuess != secretNumber){
                if (userGuess > secretNumber){
                    System.out.println("Too high");
                }
                else{
                    System.out.println("Too low");
                }
            }
            else{
                incorrectGuess = false;
                System.out.println("CORRECT!!");   
            }
        }

    }
}

File: Python Assignments/untitled folder/guess_the_number_game.py

secret_number = 7
incorrect_guess = True

while(incorrect_guess):
    user_guess = int(input("Guess!, enter a number: "))
    if user_guess != secret_number:
        if user_guess > secret_number:
            print("Too high")
        else:
            print("Too low")
    else:
        incorrect_guess = False
        print("CORRECT!!")